README.md

```diff
@@ -445,6 +447,70 @@ This repo includes a [GitHub workflow for building a docker image](./.github/wor
 
 The image includes all core WordPress files for the version specified for `WP_VERSION` in the `Dockerfile`, as well as the theme and plugin files necessary for the site. The other element required for the site to run is the database, which is excluded, since each environment should have its own database that is specified by environment variables. This allows local developers to test against local data without interfering with production or staging environments.
 
+#### Accessing the GitHub Container Registry
+
+Before you can pull the docker image, you'll need to authenticate with GitHub's Container Registry. To do that, follow these steps:
+
+1. [Generate a Personal Access Token (classic)](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens#creating-a-personal-access-token-classic) with at least `read:packages` access
+1. Copy the access token (you won't be able to see it again)
+1. Run `docker login ghcr.io` and use your GitHub username and the access token for username/password
+
+#### Running the Docker Image Locally
+
+To test the published Docker image locally, follow these steps:
```

Comment:
I think we should clarify up front that we're testing with the local development db.
```suggestion
The Docker image can be tested locally by connecting it to the local development database. To test the published Docker image locally, follow these steps:
```